src/components/modules/Contact/ContactForm/index.tsx

```diff
@@ -40,7 +40,7 @@ const ContactForm = () => {
           });
 
           setTimeout(() => resetForm(), 4000);
-        } catch (err) {
+        } catch {
```

Comment:
Fix variable is defined but never used, by updating the catch block to remove the unused `err` parameter.
```
'err' is defined but never used.eslint[@typescript-eslint/no-unused-vars](https://typescript-eslint.io/rules/no-unused-vars)
```